accepted/3.0/records/feature-specification.md

```diff
@@ -462,6 +462,27 @@ var c = cond ? a : b; // c has type `Record`.
 
 The greatest lower bound of records with different shapes is `Never`.
 
+### Combined member signatures
+
+In order to determine how a combined member signature is computed when one
+or more types in the signature is a record type, we need to extend the
+definition of the type function `NNBD_TOP_MERGE` when it is applied to a
+record type:
+
+`NNBD_TOP_MERGE` is only defined when it is applied to two types with the
+same structure, in particular: It is only defined on two record types when
```

Comment:
Instead of saying both "structure" and "shape", I'd just use "shape" consistently throughout. It's unambiguous.

Reply:
Of course, thanks!